packages/domain-picker/src/domain-picker/suggestion-item.tsx

```diff
@@ -79,9 +87,38 @@ const DomainPickerSuggestionItem: FunctionComponent< Props > = ( {
 				checked={ selected }
 			/>
 			<div className="domain-picker__suggestion-item-name">
-				<div>
+				<div className="domain-picker__suggestion-item-name-inner">
 					<span className="domain-picker__domain-name">{ domainName }</span>
-					<span className="domain-picker__domain-tld">{ domainTld }</span>
+					<span
+						className={ classnames( 'domain-picker__domain-tld', {
+							'with-margin': ! hstsRequired,
+						} ) }
+					>
+						{ domainTld }
+					</span>
+					{ hstsRequired && (
+						<InfoTooltip
+							position="middle right"
```

Comment:
I think we need to do:

```suggestion
							position={ isMobile ? "bottom center" : "middle right" }
```

Check [this](https://github.com/Automattic/wp-calypso/blob/master/client/landing/gutenboarding/onboarding-block/acquire-intent/site-title.tsx#L88) for `isMobile`.

See:
![hash-30e8ff169c0e12af03d59d406c536b0382f14f21 calypso live_new_domains(iPhone 6_7_8)](https://user-images.githubusercontent.com/17054134/93074014-0fd5b080-f684-11ea-8a0e-6bd8be81f28b.png)